bridge/build.gradle.kts

```diff
@@ -0,0 +1,51 @@
+import org.jetbrains.kotlin.gradle.dsl.JvmTarget
+
+// For more info on versioning, see the README.
+val version = "0.1.0"
+
+plugins {
+    alias(libs.plugins.android.library)
+    alias(libs.plugins.kotlin.android)
+    alias(libs.plugins.kotlin.parcelize)
+}
+
+android {
+    namespace = "com.bitwarden.bridge"
+    compileSdk = 34
+
+    defaultConfig {
+        minSdk = 28
+
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+        consumerProguardFiles("consumer-rules.pro")
+        buildConfigField("String", "VERSION", "\"$version\"")
```

Comment:
Do we need this, is the `VERSION_CODE` and the `VERSION_NAME` not good enough?

Reply:
Since this is a library not an app, we don't get that stuff. Thus, the custom build config field 👍